aten/src/ATen/native/cuda/Normalization.cuh

```diff
@@ -822,6 +887,7 @@ Tensor batch_norm_backward_elemt_cuda_template(const Tensor& grad_out_, const Te
   int tb = std::max<int>(64/tf, 1);
   dim3 blocks_trans(input.size(1), std::max<int>(1, std::min<int>((256*1024)/input.size(1),
                                                                   (input.size(0)+tb-1)/tb)));
+  blocks_trans.y = std::min(blocks_trans.y, MAX_GRID_SIZE);
```

Comment:
Yes, I think SyncBatchNorm with a large batch and only 1 channel would cause a cuda launch error. None of the tests hit this condition, but the same bug existed with the forward kernels (which use the same calculation) and one of the tests caught it there during the last PR.